Sources/Testing/Events/Recorder/Event.AdvancedConsoleOutputRecorder.swift

```diff
@@ -64,71 +146,863 @@ extension Event {
+    }
+    
+    // Format: [total] tests completed in [duration] ([pass symbol] pass: [number], [failed symbol] fail: [number], ...)
+    let passIcon = _getStatusIcon(for: .passed)
+    let failIcon = _getStatusIcon(for: .failed) 
+    let skipIcon = _getStatusIcon(for: .skipped)
+    
+    var summaryParts: [String] = []
+    if context.totalPassed > 0 {
+      summaryParts.append("\(passIcon) pass: \(context.totalPassed)")
+    }
+    if context.totalFailed > 0 {
+      summaryParts.append("\(failIcon) fail: \(context.totalFailed)")
+    }
+    if context.totalSkipped > 0 {
+      summaryParts.append("\(skipIcon) skip: \(context.totalSkipped)")
+    }
+    
+    let summaryDetails = summaryParts.joined(separator: ", ")
+    let durationText = totalDuration.isEmpty ? "" : " in \(totalDuration)"
+    output += "\(totalTests) test\(totalTests == 1 ? "" : "s") completed\(durationText) (\(summaryDetails))\n"
+    output += "\n"
+    
+    // Failed Test Details (only if there are failures)
+    let failedTests = context.testData.filter { $0.value.result == .failed }
+    if !failedTests.isEmpty {
+      output += "══════════════════════════════════════ FAILED TEST DETAILS (\(failedTests.count)) ══════════════════════════════════════\n"
+      output += "\n"
+      
+      // Iterate through all tests that recorded one or more failures
+      for (testIndex, testEntry) in failedTests.enumerated() {
+        let (testID, testData) = testEntry
+        let testNumber = testIndex + 1
+        let totalFailedTests = failedTests.count
+        
+        // Get the fully qualified test name by traversing up the hierarchy
+        let fullyQualifiedName = _getFullyQualifiedTestNameWithFile(testID: testID, context: context)
+        
+        let failureIcon = _getStatusIcon(for: .failed)
+        output += "\(failureIcon) \(fullyQualifiedName)\n"
+        
+        // Show detailed issue information with enhanced formatting
+        if !testData.issues.isEmpty {
+          for (issueIndex, issue) in testData.issues.enumerated() {
+            // 1. Error Message - Get detailed error description
+            let issueDescription = _formatDetailedIssueDescription(issue, issueIndex: issueIndex, testData: testData)
+            
+            if !issueDescription.isEmpty {
+              let errorLines = issueDescription.split(separator: "\n", omittingEmptySubsequences: false)
+              for line in errorLines {
+                output += "  \(line)\n"
+              }
+            }
+            
+            // 2. Location and Source Code Context
+            if let sourceLocation = issue.sourceLocation {
+              output += "\n"
+              output += "  Location: \(sourceLocation.fileName):\(sourceLocation.line):\(sourceLocation.column)\n"
+              
+              // 3. Source Code Context (2-3 lines before/after)
+              let codeContext = _getSourceCodeContext(for: sourceLocation)
```

Comment:
This is a cool idea in theory but as we discussed offline, it isn't possible to obtain the context of surrounding lines of code in all cases (for example, when running on a remotely-connected device) and so I'm hesitant to include this. However, I do think it would be very useful to show the unmodified source code of the `#expect` (or similar expectation) and use that as a place to denote the subexpressions and their values on subsequent lines. But for this PR, I would suggest excluding this since it won't have a consistent experience in all usage contexts.